openspec/changes/add-jsonschema-object-transforms/specs/object-projections/spec.md

```
@@ -0,0 +1,45 @@
+## ADDED Requirements
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)